libyul/backends/wasm/WasmCodeTransform.h

```diff
@@ -85,6 +85,14 @@ class WasmCodeTransform
 
 	wasm::FunctionDefinition translateFunction(yul::FunctionDefinition const& _funDef);
 
+	/// Imports an external function into the current module.
+	/// This external function is accessible within the current module via `_builtin->name` where
```

Comment:
```suggestion
	/// This external function is accessible within the current module via `_builtin->name` where
```
?

Reply:
I changed the method signature and introduced an "internal name" parameter. Maybe this makes it better understandable.